tools/kubernetes/spark-block-cleaner/src/main/scala/org/apache/kyuubi/tools/KubernetesSparkBlockCleaner.scala

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.tools
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+import java.util.concurrent.{LinkedBlockingQueue, ThreadPoolExecutor, TimeUnit}
+
+import org.apache.kyuubi.Logging
+import org.apache.kyuubi.tools.Constants._
+/*
+* Spark storage shuffle data as the following structure.
+*
+* local-dir1/
+*   blockmgr-uuid/
+*     hash-sub-dir/
+*       shuffle-data
+*       shuffle-index
+*
+* local-dir2/
+*   blockmgr-uuid/
+*     hash-sub-dir/
+*       shuffle-data
+*       shuffle-index
+*
+* ...
+*/
+object KubernetesSparkBlockCleaner extends Logging {
```

Comment:
We need a good unit test for this class, maybe a followup.

Reply:
Ok， I will commit unitTest and doc next week.